Makefile

```diff
@@ -1,7 +1,7 @@
 MODULE   = $(shell env GO111MODULE=on $(GO) list -m)
 DATE         ?= $(shell date +%FT%T%z)
 KO_DATA_PATH  = $(shell pwd)/cmd/$(TARGET)/kodata
-TARGET        = kubernetes
+TARGET        = kubernetes/operator
```

Comment:
can we keep the one word `kubernetes` or `openshift` target.. 
because, the ideas is to build for a kubernetes distro, everyting the operator has (controller, webhook, rbac...).

can we keep this as `kubernetes` and make necessary changes in the Makefile and scripts.

cc @vdemeester

Reply:
in other words, lets handle this complexity internally, withou exposing it through Make targets (script args), so that a developer who is running the Maker targets can just decide between platforms.

Reply:
Yes i m about to update PR to handle this :grin:

Reply:
Updated